### ISSUE TITLE
RetroAchievements password isn't set properly when it contains special characters

### ISSUE THREAD

**Issue by reddraggone9** [dragoonDorise/EmuDeck#251]
https://github.com/dragoonDorise/EmuDeck/blob/54c1f98e86acdf9d919dc267779635db9cc1d421/install.sh#L1726

Since the RetroAchievements password is set using sed, sed can interpret special characters in the password. In my case, my password contains `&`, and `&`s "reference the whole matched portion of the pattern space"[^1]. For a practical example of the problem, running this in a shell
```sh
curl --remote-name https://raw.githubusercontent.com/dragoonDorise/EmuDeck/main/configs/org.libretro.RetroArch/config/retroarch/retroarch.cfg
rap='abcd&1234'
raConfigFile='retroarch.cfg'
sed -i "s|cheevos_password = \"\"|cheevos_password = \"${rap}\"|g" $raConfigFile
```
results in this line in the RetroArch config file
```
cheevos_password = "abcdcheevos_password = ""1234"
```
There appear to be several characters that could cause an issue aside from `&`. [What characters do I need to escape when using sed in a sh script?](https://unix.stackexchange.com/q/32907/46690)

[^1]: https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html

**Comment by Godsbane** [comment]
I will look at implementing the solution in this SO article:
https://stackoverflow.com/questions/407523/escape-a-string-for-a-sed-replace-pattern

Thanks for reporting!

**Comment by dragoonDorise** [comment]
Fixed in 0.17.6
